app/templates/play/campaign-view.jade

```diff
@@ -90,7 +90,7 @@ if view.showAds()
                     br
                     button.btn.btn-primary.btn.btn-lg.btn-illustrated
                       span(data-i18n="play.play_classroom_version") Play Classroom Version
-        else if level.unlocksHero && !level.purchasedHero
+        else if level.unlocksHero && !level.purchasedHero && (!me.get('hourOfCode') || me.isPremium() || !level.requiresSubscription)
```

Comment:
This logic could use a comment explaining that 1) this level is hidden (the `if` is pretty far away) and 2) an explanation for what this is for. Maybe "show available hero unlocks for hidden levels"?